From acfc4983446139a849c57c8ba001eca372010ab3 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Wed, 29 Jul 2009 17:04:18 +0000
Subject: Also update Clutter 1.0 pkg-config requirement and README

We need to also change the pkg-config requirement to be 1.0.
---
diff --git a/README b/README
index 66af52b..4ec8f2d 100644
--- a/README
+++ b/README
@@ -20,7 +20,7 @@ libstartup-notification at
 http://www.freedesktop.org/software/startup-notification/ or on the
 GNOME ftp site. You also need GConf 1.2 (unless building a funky
 extra-small embedded metacity with --disable-gconf, see below).
-You need Clutter 0.9.3. You need gobject-introspection 0.6.3.
+You need Clutter 1.0. You need gobject-introspection 0.6.3.
 
 REPORTING BUGS AND SUBMITTING PATCHES
 ===
diff --git a/configure.in b/configure.in
index a1173d5..b843a68 100644
--- a/configure.in
+++ b/configure.in
@@ -227,8 +227,8 @@ else
   AC_MSG_ERROR([no. Mutter requires the Xcomposite extension to build.])
 fi
 
-CLUTTER_VERSION=0.9.3
-CLUTTER_PACKAGE=clutter-0.9
+CLUTTER_VERSION=1.0.0
+CLUTTER_PACKAGE=clutter-1.0
 AC_SUBST(CLUTTER_PACKAGE)
 if $PKG_CONFIG --atleast-version $CLUTTER_VERSION $CLUTTER_PACKAGE ; then
   MUTTER_PC_MODULES="$MUTTER_PC_MODULES $CLUTTER_PACKAGE "
--
cgit v0.8.2
